app/statistics/template.hbs

```diff
@@ -1,202 +1,209 @@
 {{#if (or (media 'isMo') (media 'isXs') (media 'isSm'))}}
   {{no-mobile-message}}
 {{else}}
-  <h1>Statistics</h1>
+  {{#if (cannot "access page")}}
+    <div class="text-center" data-test-access-forbidden>
+      <h1>Access forbidden</h1>
+      <h4>You do not have the permission to access this page</h4>
+    </div>
+  {{else}}
```

Comment:
You should make a wrapper component for this. E.g:

```hbs
{{!-- page-permission.hbs --}}

{{#if (cannot "access page")}}
  <div class="text-center" data-test-access-forbidden>
    <h1>Access forbidden</h1>
    <h4>You do not have the permission to access this page</h4>
  </div>
{{else}}
 {{yield}}
{{/if}}
```
Which you can call then like this:
```hbs
<PagePermission>
  <h1>Statistics</h1>
  ...
</PagePermission>
```